README.md

```diff
@@ -1 +1,229 @@
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# Frontend_Hello_Rails_React
+
+> This is a project to learn how to use React with Rails by Implementing a connection between a Ruby on Rails back-end and React front-end. This project contains the fronted part of it.
+
+**Hello react rails** is a simple web app to test working with rails and react in a fullstack web devevelopment project.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> These are technologies that I used.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **This app has API end points**
+- **It fetches from the API and renders the data**
+- **It renders random greetings on refreshing and navigating to the './     message**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local clone the repository https://github.com/NATASHA-ct/frontend_hello_rails_react.git
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Ruby
+- Postgresql
+- Node.js
+- Yarn
+- Rails
+
+### Setup
+
+Clone this repository to your desired folder:
+$ git clone
+
+ https://github.com/NATASHA-ct/frontend_hello_rails_react.git
+
+#### Link to backend repository
+    https://github.com/NATASHA-ct/Backend_Hello_Rails_React.git
+
+#### Link to backend repository pull request
+ 
+    https://github.com/NATASHA-ct/Backend_Hello_Rails_React/pull/1
+
+### Install
+
```

Comment:
- [ ] In this section of readme at least these commands should exist which was requested by the previous reviewer 😊

**Frontend Readme** 💄
```javascript
### Install
git clone git@github.com:NATASHA-ct/Frontend_Hello_Rails_React.git
cd Frontend_Hello_Rails_React
npm install
npm start

```
**Backend Readme** 🔧
```ruby
### Install
git clone git@github.com:NATASHA-ct/Backend_Hello_Rails_React.git
cd Backend_Hello_Rails_React
bundle install install
rails s

```